pages/wasm/index.js

```diff
@@ -0,0 +1,37 @@
+import repoLink from "../../components/repo-link.js";
+
+export default {
+  title: "WebAssembly",
+  metaDescription: "Run WebAssembly with Edge Functions.",
+  page: function () {
+    return `
+    <section>
+      <h1>WebAssembly</h1>
+      <p>You can use Edge Functions to return a JSON response by returning <code>context.json()</code> with a JavaScript object — no need to <code>JSON.stringify</code>!</p>
```

Comment:
You're still talking about .json here :D

Reply:
"I won't forget to change this," said Sean, lying to himself